quinn-proto/src/connection/streams/mod.rs

```diff
@@ -443,6 +446,24 @@ pub struct ClosedStream {
     _private: (),
 }
 
+impl ClosedStream {
+    #[doc(hidden)] // For use in quinn only
```

Comment:
Oh, this must've been why we split it out before. I don't mind this solution, though. There's also the fact that "has not been opened" in the doc comment doesn't apply at the `quinn` layer, where you can never get a handle to an unopened stream, but I don't think that's a big deal either.